### PR TITLE
Move translations from spree design

### PR DESCRIPTION
**Pull request by pmilewski**

### CODE REVIEW

File: core/config/locales/en.yml [hunk 1]
```diff
@@ -1102,7 +1217,13 @@ en:
     price_range: Price Range
     price_sack: Price Sack
     process: Process
-    product: Product
```
Comment (damianlegawiec) [inline core/config/locales/en.yml]: @pmilewski let's keep `product` key in tact

File: core/config/locales/en.yml [hunk 2]
```diff
@@ -1102,7 +1217,13 @@ en:
...
+    product:
```
Comment (damianlegawiec) [inline core/config/locales/en.yml]:
```suggestion
    pdp:
```

File: core/config/locales/en.yml [hunk 3]
```diff
@@ -604,15 +628,32 @@ en:
...
     category: Category
+    category_nav_bar:
+      get_up_to_30_percent_off: GET UP TO 30% OFF
+      jackets_and_coats: JACKET & COASTS
+      new_collection: NEW COLLECTION
+      special_offers: SPECIAL OFFERS
+      summer_2019: SUMMER 2019
     channel: Channel
     charged: Charged
-    checkout: Checkout
```
Comment (damianlegawiec) [inline core/config/locales/en.yml]: @pmilewski let's leave `checkout` in-tact

File: core/config/locales/en.yml [hunk 4]
```diff
@@ -604,15 +628,32 @@ en:
...
+    checkout:
```
Comment (damianlegawiec) [inline core/config/locales/en.yml]:
```suggestion
    checkout_page:
```

File: core/config/locales/en.yml [hunk 5]
```diff
@@ -604,15 +628,32 @@ en:
     card_number: Card Number
     card_type: Brand
     card_type_is: Card type is
-    cart: Cart
```
Comment (damianlegawiec) [inline core/config/locales/en.yml]: @pmilewski let's leave `cart` in-tact

File: core/config/locales/en.yml [hunk 6]
```diff
@@ -604,15 +628,32 @@ en:
...
+    cart:
```
Comment (damianlegawiec) [inline core/config/locales/en.yml]:
```suggestion
    cart_page:
```

**Review by damianlegawiec — CHANGES_REQUESTED** [review]

File: core/config/locales/en.yml [hunk 7]
```diff
@@ -604,15 +628,32 @@ en:
...
+      add_promo_code: ADD PROMO CODE
+      checkout: checkout
+      empty_info: "Your cart is empty.<br />There are no items in your cart."
```
Comment (przemosk) [inline core/config/locales/en.yml]:
```suggestion
      empty_info: 'Your cart is empty.<br />There are no items in your cart.'
```

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**